### ISSUE TITLE
Newest Update possibly caused forge bucket fluid transfer issue.

### ISSUE THREAD

**Issue by IBeCrayZ** [SleepyTrousers/EnderCore#53]
I opened this issue on EnderIO thinking it was caused there, however I believe it comes from the newest update which changed some fluid things.  "Added support for item stacks with fluid capabilities " With this newest version (tested on one before, works fine) I can't add any EnderIO fluids to fluid banks, inventory panels, killer joe etc.

**Comment by AnrDaemon** [comment]
Did you read the answer in https://github.com/SleepyTrousers/EnderIO/issues/3566#issuecomment-236402085 ?

**Comment by HenryLoenwind** [comment]
@CrazyPants Any idea here? I could look at this earliest Monday or Tuesday.

**Comment by CrazyPants** [comment]
How are you trying to add them? I need more detail than this.
Are you using conduits? Push/pull from a tank? Bucket in your hand?

**Comment by CrazyPants** [comment]
This is fixed in ender core build 44

**Comment by IBeCrayZ** [comment]
Sorry, was asleep during these comments. Ok so I've tested this, and for the most part it works.  So I can now place nutrient distillation, cloud seed, fire water etc into fluid tanks via right click, and can now place bucket-filled fluids into the tank to empty them (via left hand side of gui). However, I can't seem to place the normal minecraft bucket in the right hand side of the fluid tank gui to take out fluid from the tank into the bucket. This doesn't even work when trying to take water/lava from inside the tank. Furthermore, trying to right click an empty bucket onto the tank to take out the fluid doesn't fill the bucket and just empties the tank, leaving nothing.

So basically to sum it up. I can give the fluid tank fluids with both ways (via right click and gui), however can't take out fluids (via right click with empty bucket and gui)

Also, I can't directly right click on Killer Joe to add nutrient distillation via bucket and right click, and same with the inventory panel however I can't remember if you could do that before.

Fluid conduits seem to work as intended.. Tested by using both Fluid tank and Vat containing nutrient distillation into an inventory panel.

**Comment by IBeCrayZ** [comment]
Can't really do much more testing to be honest, can't think of anything else this could be.

**Comment by RainbowGravity** [comment]
Just writing to say I also have this issue and how no idea how to fix it apart from going back a few versions.

**Comment by CrazyPants** [comment]
Thanks so much for the detailed reports IBeCrayZ
I will have a fix out soon. Just FYI this is all down to forge changing the way it handles fluids, and us attempting to support both the old and new systems. In adding support for the new, I broke some of the old.

**Comment by CrazyPants** [comment]
'Furthermore, trying to right click an empty bucket onto the tank to take out the fluid doesn't fill the bucket and just empties the tank, leaving nothing'
I am guessing you where in creative mode, where this is the expected behavior.

**Comment by CrazyPants** [comment]
I have just uploaded build 45.
Hopefully this should resolve the remaining issues.
